app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -60,12 +60,12 @@ def start
     begin
       kubernetes_client.create_pod(pod)
     rescue SocketError, KubeException => e
-      msg = "pod creation for #{pod_full_name} failed: #{e}"
-      _log.info(msg)
-      return queue_signal(:abort_job, msg, "error")
+      return queue_abort("pod creation for #{pod_full_name} failed: #{e}")
```

Comment:
In [8747](https://github.com/ManageIQ/manageiq/pull/8747/files#r63435578) I suggested adding a param to `queue_abort` for an exception.  You could just pass `e` in as a param and let `queue_abort` decide how to log the exception details.